ext/rav1e.cmd

```diff
@@ -13,7 +13,7 @@
 git clone -b v0.6.6 --depth 1 https://github.com/xiph/rav1e.git
 
 cd rav1e
-cargo install cargo-c
+cargo install cargo-c --locked
```

Comment:
Nit: List the "--locked" option before "cargo-c".

Reply:
Done